dev/devicelab/test/run_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+void main() {
+  group('run.dart script', () {
+    Future<int> runScript(List<String> testNames) async {
+      List<String> options = <String>['bin/run.dart'];
+      for (String testName in testNames) {
+        options..add('-t')..add(testName);
```

Comment:
Avoided lists because our style guide enforces generic type parameters, but done anyway. Not much different in this case.